website/addons/base/views.py

```diff
@@ -1,45 +1,48 @@
+import datetime
+import httplib
 import os
 import uuid
-import httplib
```

Comment:
Please do not shuffle around import statements unless it's to solve a problem, especially on a PR like this one. Don't change it back, but please resist the impulse in the future. If there was something broken, then that's cool.